BLSQ/iaso
# POLIO-1755: Vaccine repository file filter bug

Deleted VRFs would still appear, need to only show selected file type when using filter

Related JIRA tickets : POLIO-1755

## Self proofreading checklist


## Doc

Tell us where the doc can be found (docs folder, wiki, in the code...).

## Changes
- Filter out (soft) deleted vrfs from filter results
- Only show column for selected file type

## How to test

With a polio DB:
- Delete a VRF
- Go to vaccine repository:
    - Use file type filter: only the column for selected file type
    - Deleted VRF should not be visible 

## Print screen / video


https://github.com/user-attachments/assets/84731f2e-b64e-4ba4-84eb-08838ccca1f8


## Follow the Conventional Commits specification

The **merge message** of a pull request must follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) specification.

This convention helps to automatically generate release notes.

Use lowercase for consistency.

[Example](https://github.com/BLSQ/iaso/commit/8b8d7d3064138c1e57878f17b4eb922516ab0112):

```
fix: empty instance pop up

Refs: IA-3665
```

Note that the Jira reference is preceded by a _line break_.

Both the line break and the Jira reference are entered in the _Add an optional extended description…_ field.
